dw-date-range-selection.js

```diff
@@ -33,18 +33,31 @@ export class DwDateRangeSelection extends LitElement {
     ];
   }
 
+  constructor(){
+    super();
+
+    this.noReset = true;
+    this._onFromDateChanged = this.__onFromDateChanged.bind(this);
+  }
+
+  static get properties() {
+    return {
+
+      /**
+       * Input property.
+       * Default value is true.
+       * When true, does not reset the to date if it's invalid.
+       */
+      noReset: {  type: Boolean }
```

Comment:
default value of the boolean property is always false. So set it false. by default date will be reset. You  integrator don't want to reset it. set this property to true